src/s3fs.cpp

```diff
@@ -3078,28 +3078,17 @@ static int s3fs_check_service(void)
   }
 
   S3fsCurl s3fscurl;
-  if(-1 == s3fscurl.CheckBucket()){
+  if(1 == s3fscurl.CheckBucket()){
```

Comment:
We should not change codes in curl.cpp, so I think this if-steatment codes(from L:3081 to L3084) should be removed.